viewer/src/main/webapp/viewer-html/components/EditBulk.js

```diff
@@ -0,0 +1,396 @@
+(function (Ext, undefined) {
+    Ext.define("viewer.components.EditBulk", {
+        extend: "viewer.components.Edit",
+        lastClickCoords: null,
+        selectedFeatures: null,
+        originalFormProperties: null,
+        storeOriginalFormProperties: function () {
+            var me = this;
+            me.originalFormProperties = {};
+            var attributeNames = me.getListOfAttributeNamesThatAreMappedToFormFields();
+            attributeNames.forEach(function (attributeName) {
+                var input = me.inputContainer.getForm().findField(attributeName);
+                me.originalFormProperties[attributeName] = {
+                    allowBlank: input.allowBlank,
+                    emptyText: input.emptyText
+                }
+            });
+        },
+        initAttributeInputs: function (appLayer) {
+            viewer.components.EditBulk.superclass.initAttributeInputs.call(this, appLayer);
+            this.storeOriginalFormProperties();
+            this.selectedFeatures = new SelectedFeatures(
+                this.config.viewerController.mapComponent.getMap(),
+                this.getListOfAttributeNamesThatAreMappedToFormFields()
+            );
+        },
+        makeFormFieldMultiple: function (attributeName) {
+            var field = this.inputContainer.getForm().findField(attributeName);
+            var placeholder = this.buildPlaceholderForAttributeWithMultipleValues(attributeName);
+            field.setEmptyText(placeholder);
+            field.allowBlank = true;
+            field.setValue('');
+            field.validate();
+        },
+        buildPlaceholderForAttributeWithMultipleValues: function (attributeName) {
+            var LEEG = '<leeg>';
+            var MEERDERE_WAARDES = 'Meerdere';
```

Comment:
these values need to be localizable using `i18next.t()`English and Dutch  resource values can then be added to the respective properties files in https://github.com/flamingo-geocms/flamingo/tree/master/viewer/src/main/resources